rust-runtime/aws-smithy-http-server/examples/pokemon-service/src/main.rs

```diff
@@ -47,10 +52,21 @@ pub async fn main() {
         // implementation.
         .into();
 
+    // New builder
+    let get_pokemon_species = GetPokemonSpecies::from_handler(get_pokemon_species);
```

Comment:
I think this would give an error saying that `Op` and `Exts` are unconstrained.

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d42bc87814ba44f09e5df4464f084fbd

Reply:
This is a bit closer but `Ext` is still unconstrained unfortunately.

```rust
impl<Op, H, Exts> From<H> for Operation<IntoService<Op, H>>
where
    Op: OperationShape,
    H: Handler<Op, Exts>,
{
    fn from(h: H) -> Self {
        todo!()
    }
}
```

Reply:
If `Exts` was an associated type on `Handler` then this would work and worth experimenting with. It might have some knock on effects elsewhere though.